config/environments/development.rb

```diff
@@ -14,21 +12,27 @@
   # Show full error reports.
   config.consider_all_requests_local = true
 
+  # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
-  if Rails.root.join("tmp/caching-dev.txt").exist?
+  if Rails.root.join('tmp', 'caching-dev.txt').exist?
     config.action_controller.perform_caching = true
+    config.action_controller.enable_fragment_cache_logging = true
 
     config.cache_store = :memory_store
     config.public_file_server.headers = {
-      "Cache-Control" => "public, max-age=#{2.days.to_i}"
+      'Cache-Control' => "public, max-age=#{2.days.to_i}"
     }
   else
     config.action_controller.perform_caching = false
 
     config.cache_store = :null_store
   end
 
-  config.action_mailer.raise_delivery_errors = true
+  # Store uploaded files on the local file system (see config/storage.yml for options).
+  config.active_storage.service = :local
+
+  # Don't care if the mailer can't send.
+  config.action_mailer.raise_delivery_errors = false
```

Comment:
Is this a Rails 6 related change, or unrelated?  I'm just curious.

Reply:
It's based on the `app:update` default `development.rb`. I'm not sure why we had it set to `true`.